pkg/cmd/extension/ext_tmpls/goBinWorkflow.yml

```diff
@@ -5,10 +5,15 @@ on:
       - "v*"
 permissions:
   contents: write
+  id-token: write
+  attestations: write
```

Comment:
After testing, I found these permissions were required for the workflow to successfully and create attestations.